### PR TITLE
Scroll integration to main branch

### PR DESCRIPTION
**Pull request by alhonaut**

### CODE REVIEW

**Comment by Horlarmmy** [comment]
I'm seeing a gig contract submodule there, can you fix that?

File: gig-contracts
Comment (Horlarmmy) [inline gig-contracts:1]: Here, it looks like its linking to a repo

**Comment by Horlarmmy** [comment]
There is a conflict, please fix that

**Review by Horlarmmy — APPROVED** [review]
Great, but i still see the gig contract submodule

**Comment by alhonaut** [comment]
> Great, but i still see the gig contract submodule

Yea, strange, will try to remove it

**Review by mgroovyank — APPROVED** [review]

**Pull request merged**